packages/react-aria-components/docs/FileTrigger.mdx

```diff
@@ -122,14 +122,35 @@ A file trigger can accept multiple files by passsing the `allowsMultiple` proper
 
 ## Directory selection
 
-To enable selecting directories instead of files, use the `directory` property. 
+To enable selecting directories instead of files, use the `acceptDirectory` property. 
```

Comment:
Lets add a blurb at the end of the next paragraph about how support will vary from browser to browser.